src/components/Tasks/Toolbar.tsx

```diff
@@ -20,23 +24,66 @@ import { StoreState } from '../../store';
 
 import Menu from '../../widgets/Menu';
 
+const transitionTimeout = 500;
+
+const transitionStyles = {
+  entering: { visibility: 'visible', width: '100%', overflow: 'hidden' },
+  entered: { visibility: 'visible', width: '100%' },
+  exiting: { visibility: 'visible', width: '0%', overflow: 'hidden' },
+  exited: { visibility: 'hidden', width: '0%' },
+};
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
All of these styles feel like a hack to me. Why do you need such complex styling? Doesn't the transition component already cover the transition stuff? Is autocomplete still there? I didn't see.